docs/event-bus/docs/050-subscription-updates.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Subscription updates
+type: Details
+---
+
+The subscription crd can be updated using kubectl:
+
+`kubectl edit crd subscriptions.eventing.kyma.cx`
+
+Event-bus reacts on subscription crd changes, and it updates the corresponding NATS-Streaming subscription accordingly.
```

Comment:
`The Event Bus reacts to the changes in subscription CRD, and updates the corresponding NATS-Streaming subscription accordingly.`